CONTRIBUTING.md

```diff
@@ -50,7 +50,7 @@ You need the following software to contribute to temBoard:
 
 ## Setup Development
 
-Running development version of UI and agent requires two terminals, one for the
+Running development version of UI and agent requires two terminals, one for each.
 UI and one for the agent.
```

Comment:
Sorry for the confusion but "UI and one for the agent." was meant to be removed.